### PR TITLE
Olha Mazurenko

### PR DESCRIPTION
**Pull request by OlhaMazurenko**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      }
+      this.items.sort((x, y) => {
+        if(x.priority > y.priority) return -1;
+        if(x.priority < y.priority) return 1;
+        if(x.priority === y.priority) return 0;
+      });
+      return this.items[0];
+    } catch(e) {
+    console.log(e);
+      return false;
```
Comment (gestap) [inline main.js]: you need to `...throw an error` instead return false

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: main.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    return this.itemsID;
+  }
+  removeItem(id) {
+    delete this.items[id];
+    return this.items[id] 
+             ? true
+             : false;
+  }
+  getItem(id) {
+    return this.items[id] ? this.items[id] : null;
```
Comment (ost-ptk) [inline main.js:29]:
```suggestion
    return this.items[id] || null;
```

File: main.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  }
+  addItem(title, priority) {
+    this.itemsID++;
+    this.items[this.itemsID] = {title, priority};
+
+    return this.itemsID;
+  }
+  removeItem(id) {
+    delete this.items[id];
+    return this.items[id]
```
Comment (ost-ptk) [inline main.js:24]:
```suggestion
    return !!this.items[id] 
```

File: main.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  addItem(title, priority) {
+    this.itemsID++;
+    this.items[this.itemsID] = {title, priority};
+
+    return this.itemsID;
+  }
+  removeItem(id) {
+    delete this.items[id];
+    return this.items[id] 
+             ? true
```
Comment (ost-ptk) [inline main.js:25]:
```suggestion
```

File: main.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+             : false;
```
Comment (ost-ptk) [inline main.js:26]:
```suggestion
```

**Pull request closed without merging**